nengo/connection.py

```diff
@@ -347,6 +346,9 @@ def size_out(self):
 
     @property
     def _label(self):
```

Comment:
Yeah, makes sense! It's just here as a helper for `__str__` and `__repr__` anyhow. Think I'll go with `_str`.